### PR TITLE
update content/ja/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html

### LINKED ISSUE #21168: content/ja/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html is outdated
**This is a Feature Request**

**What would you like to be added**
Update content/ja/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html to follow the en updates.

**Why is this needed**
content/ja/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html is outdated.

**Comments**
/language ja
/assign
diff
https://gist.github.com/oke-py/00d473105d39d586a252b4653a1621c2

### PR DESCRIPTION
**Pull request by oke-py**
fixes #21168

### CODE REVIEW

File: content/ja/docs/concepts/workloads/pods/pod-overview.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ card:
 
 
 {{% capture body %}}
-## Podについて理解する
```
Comment (sftim) [inline content/ja/docs/concepts/workloads/pods/pod-overview.md:16]: It's probably OK to leave the original line. There's no requirement for the fragment identifiers to be in English.

Comment (nasa9084) [reply content/ja/docs/concepts/workloads/pods/pod-overview.md:16]: If it is left the original line, the link to here need to contains Japanese characters, which will encoded and hard to read in browser's address bar

**Pull request merged**